flake.nix

```
@@ -131,6 +134,18 @@
           services = {
             openssh.enable = true;
             openssh.permitRootLogin = "prohibit-password";
+            logrotate = {
+              enable = true;
+              settings.${binplz-log-dir + "/*.log"} = {
```

Oh neat.  I thought when doing this you had to quote the key like:

```suggestion
              settings."${binplz-log-dir + "/*.log"}" = {
```

But it looks like you don't have to!